doc/ref_model/chapters/chapter03.md

```diff
@@ -214,10 +214,14 @@ The hardware infrastructure manager needs to support the following functional as
 <a name="3.4"></a>
 ## 3.4 Hardware Infrastructure Resources
 
-The physical compute, storage and network resources serve as the foundation of the cloud infrastructure. They are as such not directly exposed to the workloads (VNFs/CNFs).
+Compute, Storage and Network resources serve as the foundation of the cloud infrastructure. They are exposed to and used by a set of networked Host Operating Systems in a cluster that normally handles the Virtualization Layer offering Virtual Machines or Containers where the application workloads (VNFs/CNFs) runs.
 
-<p align="center"><img src="../figures/ch03-model-physical-resources.png" alt="Hardware Infrastructure Resources" Title="Hardware Infrastructure Resources" width="65%"/></p>
-<p align="center"><b>Figure 3-5:</b> Hardware Infrastructure Resources</p>
+<p align="center"><img src="../figures/ch03-model-hardware-resources.png" alt="Cloud Infrastructure Hardware Resources" Title="Cloud Infrastructure Hardware Resources" width="65%"/></p>
+<p align="center"><b>Figure 3-5:</b> Cloud Infrastructure Hardware Resources</p>
+
+In managed Hardware Infrastructure systems, these consumable Compute, Storage and Network resources can be provisioned through operator commands or through software APIs.  There is a need to distinguish between these consumable resources, that are treated as leased resources, from the actual physical hardware resources that are installed in the data center. For this purpose, the hardware resource layer is conceptually split into a Logical Resource Layer that surfaces the consumable resources to the software layer above, and the Physical Resource Layer that is operated and managed by the Data Center Operations team from the HW Infrastructure Management functions.
```

Comment:
```suggestion
In managed Cloud Infrastructure systems, these consumable Compute, Storage and Network resources can be provisioned through operator commands or through software APIs.  There is a need to distinguish between these consumable resources, that are treated as leased resources, from the actual physical hardware resources that are installed in the data center. For this purpose, the cloud infrastructure resource layer is conceptually split into a Logical Resource Layer that surfaces the consumable resources to the software layer above, and the Physical Resource Layer that is operated and managed by the Data Center Operations team using HW Infrastructure Management functions.
```

Reply:
I think we are blurring the management aspects of a clear separation between the Hardware Infrastructure resources as being Cloud Infrastructure Resources. Already now there are many types of resources that we have to manage.
The Hardware Infrastructure Layer has both (internal) Physical resources and the (externally exposed) consumable resources (Compute, Storage and Network that the Virtualization Infrastructure Layer are exposed to and by that consumes).
The Virtualization Infrastructure Layer instances has VM or Container with Network and Storage resources offered to their Tenants.

Reply:
Same as above, I propose to use consistent wording, not more explanation.

Reply:
@ulikleber, the issue here is that in this PR we are really talking about hardware and its management as the title itself says. Hence, replacing the word "hardware" by "infrastructure" is not appropriate in this context, as I think is what @TFredberg is saying.  I am not sure what is wrong in using the term "hardware" when we are talking about hardware :) Am I missing something?  How can we move on with this?

Reply:
I am not concerned about using the term hardware. But the term "hardware infrastructure" also relates to equipment practice, power supply, cabinets and air-conditioning etc. Thus we are talking about the hardware of the cloud infrastructure, not about the management of fans and such. Therefore I suggested the term "cloud infrastructure **hardware**". It is not a big issue. If I am the only one concerned about "hardware infrastructure", let's keep it as is.